### PR TITLE
fix: make bleak a dev requirement

### PR DESCRIPTION
**Pull request by bdraco**
We only need bleak for type checking

https://github.com/home-assistant/core/issues/83722

### CODE REVIEW

**Review by rytilahti — APPROVED** [review]
Looks good to me on the surface :+1:

**Comment by bdraco** [comment]
thanks for the second set of 👀

File: src/home_assistant_bluetooth/models.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 """The bluetooth integration service info."""
 
-from typing import Any, Dict, Final, List, Optional, Type, TypeVar
+from typing import TYPE_CHECKING, Any, Dict, Final, List, Optional, Type, TypeVar
 
 from bleak.backends._manufacturers import MANUFACTURERS
```
Comment (lipov3cz3k) [inline src/home_assistant_bluetooth/models.py:5]: This import still requires `bleak`, so it cannot be just dev-dependency.

Comment (bdraco) [reply src/home_assistant_bluetooth/models.py:5]: fixed in https://github.com/home-assistant-libs/home-assistant-bluetooth/pull/17

**Pull request merged**